build/fuchsia/fidl_gen_cpp.py

```diff
@@ -40,7 +40,7 @@ def main():
   parser.add_argument('--sdk-base', dest='sdk_base', action='store', required=True)
   parser.add_argument('--root', dest='root', action='store', required=True)
   parser.add_argument('--json', dest='json', action='store', required=True)
-  parser.add_argument('--fidlgen-root', dest='fidlgen_root', action='store', required=True)
+  parser.add_argument('--fidlgen-output-root', dest='fidlgen_output_root', action='store', required=False)
```

Comment:
No action required, but it seems weird that fidlgen_bin is always required even though it won't be used when fidlgen_output_root is not set.

Reply:
noted. I've changed the code to consider this.